lighthouse-core/test/fixtures/traces/frame-metrics-m90.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
Keeping `frame-metrics-m89.json` around so we don't have to change expectations for LCP-AF and FCP-AF everywhere.